### DISCUSSION TITLE
Does kwok support simulating GPU nodes ?

### DISCUSSION THREAD

**Discussion by FuPengCheng1** [kubernetes-sigs/kwok#618] [Q&A] [answered]
### What would you like to be added?

I have not found docs about simulating gpu nodes by kwok. Does kwok support it?

### Why is this needed?

GPU nodes are also widely used online like cpu&mem nodes.  I think support simulating gpu nodes is also needed.

**Comment by wzshiming** [comment, marked as answer]
In a kwok context, Nodes and Pods are nothing but pure API objects so feel free to mutate their API specs to do whatever simulation or testing you want.

**Comment by FuPengCheng1** [comment]
Thanks. I have succcessfully simulated gpu nodes and deploy fake-pods on it.
